conda_recipes/cinema4d-openjd/recipe/activate.sh

```diff
@@ -0,0 +1,13 @@
+export DEADLINE_CLOUD_PYTHONPATH="$CONDA_PREFIX\\Lib\\site-packages"
+export DEADLINE_CINEMA4D_EXE="$CONDA_PREFIX\\c4d\\Commandline.exe"
+
+# We need to add additional paths for pywin32 dependency.
+export C4DPYTHONPATH311="$CONDA_PREFIX\\Lib\\site-packages;$CONDA_PREFIX\\Lib\\site-packages\\win32;$CONDA_PREFIX\\Lib\\site-packages\\Pythonwin;$CONDA_PREFIX\\Lib\\site-packages\\win32\\Lib"
```

Comment:
Maybe this could be incorporated into the Cinema4D conda build recipe instead of the adaptor?

Reply:
Hmm, the pywin32 dependencies are added by the `openjd-adaptor-runtime` which is installed during the installation of this adaptor. So should we still try to build this in the Cinema 4D conda build recipe?